docs/group_desc_en.md

```diff
@@ -1,20 +1,20 @@
 `Zencoders` is a group of software developers and informatics enthusiasts
-cooperating to realize common projects or just to encourage the flow of ideas.  
-`Zencoders` group leverages knowledge sharing amongst its members to develop
+cooperating to realize common projects or encourage the flow of ideas.  
+The `Zencoders` group leverages knowledge sharing amongst its members to develop
 innovating ideas.
 
 Each group member could contribute to any project hosted in `Zencoders` open
 repositories or start a new project.  
 `Zencoders` do not adopt nor push towards a specific technology, but source
-code of the projects will always be *open* and released with open liceses [GPL and
+code of the projects will always be *open* and released with open licenses [GPL and
 GPL-compatible, BSD, MIT, Apache, Mozilla, Eclipse, DTFYW].  
 It follows that changes and contributions (they're welcome) may come from anyone
 interested in the project, but each project has a limited number of maintainers,
 belonging to `Zencoders`.
-Anyone willing to contribute to a `Zencoders` project on a regular basis may ask
-to join the group, agreeing its rules, and then ask directly the project maintainer
-to have the access granted.
+Anyone willing to contribute to a `Zencoders` project on a regular basis should ask
```

Comment:
Here I tried to kept separate the group joining aspect and the project's permissions one. If you find a way to do that I'd really appreciate that.